plugins/tracker-resources/src/components/SetParentIssueActionPopup.svelte

```diff
@@ -95,7 +95,6 @@
   {ignoreObjects}
   shadows={true}
   {width}
-  searchField="title"
```

Comment:
May be noSearchField is required to be added here?

Reply:
Yes, somehow removed it before committing. Fixed